avm/res/cache/redis-enterprise/main.bicep

```diff
@@ -0,0 +1,649 @@
+            }
+          ]
+        : [],
+      contains(secretsExportConfiguration!, 'secondaryAccessKeyName')
+        ? [
+            {
+              name: secretsExportConfiguration!.secondaryAccessKeyName
+              value: redisDatabase.listKeys().secondaryKey
+            }
+          ]
+        : [],
+      contains(secretsExportConfiguration!, 'secondaryConnectionStringName')
+        ? [
+            {
+              name: secretsExportConfiguration!.secondaryConnectionStringName
+              value: '${redisDatabase.properties.clientProtocol == 'Plaintext' ? 'redis://' : 'rediss://' }:${redisDatabase.listKeys().secondaryKey}@${redisCluster.properties.hostName}:${redisDatabase.properties.port}'
+            }
+          ]
+        : []
+    )
+  }
+}
+
+// ============ //
+// Outputs      //
+// ============ //
+
+@description('The name of the Redis cluster.')
+output name string = redisCluster.name
+
+@description('The resource ID of the Redis cluster.')
+output resourceId string = redisCluster.id
+
+@description('The name of the Redis database.')
+output dbName string = redisDatabase.name
+
+@description('The resource ID of the database.')
+output dbResourceId string = redisDatabase.id
+
+@description('The name of the resource group the Redis resource was created in.')
+output resourceGroupName string = resourceGroup().name
+
+@description('The Redis endpoint.')
+output endpoint string = '${redisCluster.properties.hostName}:${redisDatabase.properties.port}'
+
+@description('The location the resource was deployed into.')
+output location string = redisCluster.location
+
+@description('The private endpoints of the Redis resource.')
+output privateEndpoints array = [
+  for (pe, i) in (!empty(privateEndpoints) ? array(privateEndpoints) : []): {
+    name: redisEnterprise_privateEndpoints[i].outputs.name
+    resourceId: redisEnterprise_privateEndpoints[i].outputs.resourceId
+    groupId: redisEnterprise_privateEndpoints[i].outputs.groupId
+    customDnsConfig: redisEnterprise_privateEndpoints[i].outputs.customDnsConfig
+    networkInterfaceResourceIds: redisEnterprise_privateEndpoints[i].outputs.networkInterfaceResourceIds
+  }
+]
+
+import { secretsOutputType } from 'br/public:avm/utl/types/avm-common-types:0.4.1'
+@description('A hashtable of references to the secrets exported to the provided Key Vault. The key of each reference is each secret\'s name.')
+output exportedSecrets secretsOutputType = (secretsExportConfiguration != null)
+  ? toObject(secretsExport.outputs.secretsSet, secret => last(split(secret.secretResourceId, '/')), secret => secret)
+  : {}
+
+// =============== //
+//   Definitions   //
+// =============== //
+
+type disabledPersistenceType = {
+  type: 'disabled'
+}
+
+type aofPersistenceType = {
+  type: 'aof'
+  frequency: '1s' | 'always'
+}
+
+type rdbPersistenceType = {
+  type: 'rdb'
+  frequency: '1h' | '6h' | '12h'
+}
```

Comment:
Missing `@description('(...)')` for the properties

Reply:
Added

Reply:
Great, thanks. Resolving...